unlock-js/src/v13/configureUnlock.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Creates a lock on behalf of the user, using version v0
```

Comment:
Does this comment reflect the code below? Is there a reason why we would be using v0 here rather than v13?

Reply:
ha! Wrong comments. Let me fix.